cephfs/admin/bytecount.go

```diff
@@ -0,0 +1,37 @@
+// +build !luminous,!mimic
+
+package admin
+
+// ByteCount represents the size of a volume in bytes.
+type ByteCount uint64
```

Comment:
Is this really useful? Passing a `uint64` from applications to this package seems easier... Maybe use this internally only?

Reply:
This allows a couple nice things:
* Its more "self documenting" and acts reminder that the unit is in bytes. When I work on the heketi project this is a constant source of irritation because it doesn't use bytes, some are GiB and others KiB. Using an explicit type avoid this issue
*  Using a type allows us to add methods that allow this type to satisfy interfaces. See how resize allows a special "Infinite" constant or a byte value
* It allows us to more easily cope with the fact some of the ceph calls take strings rather than integers. We use this type consistently in our functions and then we can use it to convert to whatever ceph really wants on the backend.